pupa/tests/importers/test_bill_importer.py

```diff
@@ -50,7 +51,15 @@ def test_full_bill():
     # import bill
     oi = OrganizationImporter('jid')
     oi.import_data([org.as_dict(), com.as_dict()])
-    BillImporter('jid', oi).import_data([oldbill.as_dict(), bill.as_dict()])
+
+    pi = PersonImporter('jid')
+    pi.json_to_db_id['person-id'] = 'person-id'
```

Comment:
I wasn't clear on why the person couldn't just be imported through Pupa's API (instead of mucking in its implementation details), so a recent PR imports the person first: https://github.com/jpmckinney/pupa/commit/69089a55588f4f010bef483898e79e4f5c30ad45